### ISSUE TITLE
unable to establish a call 488 not acceptable here

### ISSUE THREAD

**Issue by danielsender** [tieto/sipe#148]
I'm running Pidgin 3.0.0devel on an Ubuntu 16.04 system. Texting works fine with buddies in skype for business, but when I try to establish a video/audio or audio conversation with a buddy is when I have a problem. Initially a window with a keypad opens up with two video screens, in the smaller one I can see the local video and the bigger one is black. After a couple of seconds that window disappears and it comes back again for another couple of seconds. Finally I get another window with the message:
*unable to establish a call
488 not acceptable here."

I'm attaching a log file. I'll appreciate any help.

Thanks.
[pidgin.log.gz](https://github.com/tieto/sipe/files/1081802/pidgin.log.gz)

**Comment by xhaakon** [comment]
```
SIP/2.0 488 Not Acceptable Here
...
Call-ID: C11DgA09Aa8391iAA9Am808CtF3EDb5E06x7173x
CSeq: 1 INVITE
User-Agent: UCCAPI/16.0.7967.5277 OC/16.0.7967.2139 (Skype for Business)
ms-client-diagnostics: 52063;reason="Unsupported session description"
...
```

Have you tried to establish audio-only call? You should see both `Audio Call` and `Audio/Video Call` options when right-clicking on a buddy.

**Comment by danielsender** [comment]
Yes, I tried both with the same result. I see the audio as well as audio/video options after I press the media button.

**Comment by danielsender** [comment]
Now I just have a question: was anybody successful in placing an audio or video/audio call to someone on skype for business? Or definitively doesn't work so I should just quit?

**Comment by rbeldin** [comment]
I can make audio calls TO people and attend conference calls and get shared desktops and share with others. 

As far as I can tell, people with 'real' Skype for Business cannot
call me, but I can call them.  I always get a popup that says
'error with your microphone' when someone tries to call me.   People
who have pidgin sipe can call me however. 

I can't tell about video, because of my ancient laptop, which lacks video.

**Comment by rbeldin** [comment]
Only with this ppa by the way: 

/etc/apt/sources.list.d/sipe-collab-ubuntu-ppa-zesty.list:deb http://ppa.launchpad.net/sipe-collab/ppa/ubuntu zesty main

I've had this working since 14.04 and now on 17.04.

**Comment by danielsender** [comment]
I'm on 16.04 . I upgrade also on sipe-collab's ppa. There must be some glitch in my system that creates that error.

Thanks

**Comment by moj0-bf** [comment]
I am having the same issue. I can join conferences with full features. However I cannot make outbound audio or audio/video calls with the above error. I also cannot accept audio or audio/video calls from others with the microphone error.

Linux Mint 18.3
Kernel 4.13.0-21
Pidgin 3.0.0devel and SIPE 1.23.0 from the PPA
